src/main/scala/cromwell/engine/db/slick/WorkflowExecutionComponent.scala

```diff
@@ -59,4 +64,34 @@ trait WorkflowExecutionComponent {
       if workflowExecution.status inSet statuses
     } yield workflowExecution
   }
+
+  /**
+   * Query workflow execution using the filter criteria encapsulated by the `WorkflowQueryParameters`.
+   */
+  def queryWorkflowExecutions(queryParameters: WorkflowQueryParameters) = {
+    val Include: Rep[Boolean] = true
```

Comment:
Currently there is no companion object, creating a new companion object to hold these breaks the inheritance/import from the `slick.driver.JdbcProfile.API` trait that provides the implicits for these conversions.  

I'm going to lowercase these.